addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -99,7 +107,33 @@ def _l10n_it_edi_check_invoice_configuration(self, invoice):
         if not buyer.vat and not buyer.l10n_it_codice_fiscale and buyer.country_id.code == 'IT':
             errors.append(_("The buyer, %s, or his company must have either a VAT number either a tax code (Codice Fiscale).", buyer.display_name))
 
-        # <1.4.2>
+        # <2.2.1>
+        for invoice_line in invoice.invoice_line_ids:
+            if not invoice_line.display_type and len(invoice_line.tax_ids) != 1:
+                raise UserError(_("You must select one and only one tax by line."))
+
+        for tax_line in invoice.line_ids.filtered(lambda line: line.tax_line_id):
+            if not tax_line.tax_line_id.l10n_it_kind_exoneration and tax_line.tax_line_id.amount == 0:
+                errors.append(_("%s has an amount of 0.0, you must indicate the kind of exoneration.", tax_line.name))
+
+        return errors
+
+    def _l10n_it_edi_check_simplified_invoice_configuration(self, invoice):
+        errors = self._l10n_it_edi_check_buyer_invoice_configuration(invoice)
+        template = self.env.ref('l10n_it_edi.account_invoice_it_simplified_FatturaPA_export', raise_if_not_found=False)
+
+        if errors and template:
+            buyer = invoice.commercial_partner_id
+            if (buyer.l10n_it_codice_fiscale or buyer.vat[:2].upper() == 'IT') and invoice.amount_total <= 400:
```

Comment:
it is the amount tax included that counts?  Were not we going to put extra checks here on the code of the country_id?

Reply:
I'm not sure it can be this way, since the vat number is not always a codice fiscale.
(sometimes a company has a codice fiscale that differs to their vat number)
I think it should be something like:
if buyer codice fiscale and partner country is Italy and amount total is less than 400

Reply:
I think you may have misunderstood, to make a simplified invoice we must either have the VAT number or the Codice fiscale. Hence, currently it checks if there is either a codice fiscale or the vat starts with IT as the country may not be filled

Reply:
You're correct. But it still seems strange to me to be utilising these fields instead of the country_id. 
I think there could be a scenario for which a client might have a codice fiscale (as part of their contact information, remember that a codice fiscale can be for an individual too), but reside outside of Italy. 

Although the country may not be filled, I think we can insist that it should be. It is already required information for the ordinary invoice.

What do you think @jco-odoo ?

Reply:
We are using the country_id.  It is just that it might not be filled in.  (which I prefer to keep, so the user can just fill in the codice fiscale or vat and it would work.